source/agora/test/NetworkManager.d

```diff
@@ -151,11 +158,20 @@ unittest
 
     Transaction[][] block_txes; /// per-block array of transactions (genesis not included)
     Transaction[] last_txs;
-    foreach (block_idx; 0 .. 10)  // create 10 blocks
+
+    // create genesis block
+    if (!last_txs.length)
     {
-        // create enough tx's for a single block
-        auto txs = makeChainedTransactions(WK.Keys.Genesis, last_txs, 1);
+        last_txs = genesisSpendable().map!(txb => txb.sign()).array();
+        last_txs.each!(tx => node_validator.putTransaction(tx));
+        network.expectBlock([node_validator], Height(1), 4.seconds);
+        block_txes ~= last_txs.sort.array;
+    }
 
+    foreach (block_idx; 1 .. 10)  // create a total of 10 blocks
```

Comment:
```suggestion
    foreach (block_idx; 1 .. 10)  // create 9 additional blocks
```

Because this loop creates 9 blocks (even if the `if` block before created 1).